kafka-connect-hazelcast/src/main/scala/com/datamountaineer/streamreactor/connect/hazelcast/writers/HazelCastWriter.scala

```diff
@@ -78,8 +71,8 @@ class HazelCastWriter(client: HazelcastInstance, settings: HazelCastSinkSettings
       logger.debug("No records received.")
     } else {
       logger.debug(s"Received ${records.size} records.")
-      val converted = records.map(r => (r.topic, convert(r)))
-      converted.foreach({
+      val grouped = records.map(r => (r.topic, r))
+      grouped.foreach({
         case (topic, payload) =>
           val writer = writers.get(topic)
           val t = Try(writer.foreach(w => w.write(payload)))
```

Comment:
why are you swallowing the error. furthermore, say record 1 of 1M fails, the remaining 999999 are not tried then.